04 Strategy Library/230 Fama French Five Factors/03 Algorithm.html

```diff
@@ -0,0 +1,66 @@
+<p>
+    Taking inspiration from the Fama French five-factor model, we can develop a multi-factor stock selection strategy that focuses on five factors: size, value, quality, profitability, and investment pattern. 
+</p>
+<p>
+    In the following backtest,  we use the terms TotalEquity, BookValuePerShare, OperationProfitMargin, ROE, and TotalAssetsGrowth to account for the five factors, respectively. We then calculate a custom ranking metric for each stock using these five terms. Our algorithm will long the five stocks with the highest scores and short the five stocks with the lowest scores.
```

Comment:
Change to "Our algorithm will _go_ long _in_ the five stocks...". Long doesn't quite work as a verb like short does